webpack/assets/javascripts/react_app/routes/Models/ModelsPage/components/ModelsPageContent.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ModelsTable from '../../../../components/ModelsTable';
+import Pagination from '../../../../components/Pagination/PaginationWrapper';
+
+import LoadingPage from '../../../common/LoadingPage';
+import { withRenderHandler } from '../../../../common/HOC';
+
+const ModelsPageContent = ({
+  models,
+  search,
+  sort,
+  fetchAndPush,
+  itemCount,
+  page,
+  perPage,
+}) => (
+  <React.Fragment>
+    <ModelsTable
+      results={models}
+      search={search}
+      sortBy={sort.by}
+      sortOrder={sort.order}
+      getTableItems={fetchAndPush}
+    />
+    <Pagination
```

Comment:
I noticed a margin between the table and the pagination component, is it intentional?

Reply:
No, not intentional, will fix.